.github/workflows/build.yml

```diff
@@ -479,9 +494,11 @@ jobs:
             swiftinstaller-macos-64-${{ needs.preBuild.outputs.version }}.dmg
             xswiftbus-fat-allos-${{ needs.preBuild.outputs.version }}.7z
             swiftsymbols-allos-${{ needs.preBuild.outputs.version }}:swiftsymbols
+      - name: Publish doxygen
+        uses: actions/deploy-pages@v3
```

Comment:
From the [``actions/deploy-pages`` documentation](https://github.com/actions/deploy-pages/tree/main?tab=readme-ov-file#security-considerations):
> The deployment should target the ``github-pages`` environment

This environment should be added to the job to be consistent.

``actions/deploy-pages`` will automatically create the page deployment in the  ``github-pages`` environment as this is the default parameter for the [API call](https://docs.github.com/en/rest/pages/pages?apiVersion=2022-11-28#create-a-github-pages-deployment) that is internally called by the Action.